<launch>
    <node pkg="mimi_common_pkg" name"enter_the_room" type="enter_the_room.py">
    </node>
</launch>
